wordle-web/src/components/ValidWords.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <v-row justify="center">
```

Comment:
I would suggest putting this v-row outside of this component. Just keeping this a dialog.